iSeeOnto/.htaccess

```diff
@@ -0,0 +1,53 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+# Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+
+RewriteEngine on
+
+#Rewrite rules for aimodel ontology
+RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+RewriteRule ^aimodel$ https://isee4xai.github.io/iSeeOnto/aimodel.rdf [R=303,NE,L]
+#default response: owl
+RewriteRule ^aimodel$ https://isee4xai.github.io/iSeeOnto/aimodel.rdf [R=303,NE,L]
```

Comment:
It looks like all these rule blocks could be simplified.  Why is the RewriteCond needed if it will go to the .rdf either way?

Reply:
In this case it may not be needed. Maybe in the future they want to support other serializations?

Reply:
They were left in from the files we based these on, as in the future we are planning on having HTML and other serialisations. I've reduce it to the default response for now and will add these later when they are available.